src/time_integration/methods_SSP.jl

```diff
@@ -82,8 +88,26 @@ mutable struct SimpleIntegratorSSP{RealT <: Real, uType, Params, Sol, F, Alg,
     alg::Alg
     opts::SimpleIntegratorSSPOptions
     finalstep::Bool # added for convenience
+    dtchangeable::Bool
+    force_stepfail::Bool
 end
 
+"""
+    add_tstop!(integrator::SimpleIntegratorSSP, t)
+Add
+"""
```

Comment:
?

Reply:
I added the missing docstring in df34188